packages/medusa/src/loaders/index.ts

```diff
@@ -45,45 +40,6 @@ type Options = {
   isTest: boolean
 }
 
-async function loadLegacyModulesEntities(configModules, container) {
```

Comment:
I already removed all of this here: https://github.com/medusajs/medusa/pull/7295 so there might be some rebase conflict resolution to do